lua/Dockerfile

```
@@ -1,7 +1,16 @@
-FROM esolang/lua:2.4.0
+FROM mileschou/lua:5.4
 
 # Set the working directory inside the container
 WORKDIR /app
 
 # Copy the current directory contents into the container at /app
 COPY ./ /app
+
+# Install Lua and luarocks
+RUN apt-get update && \
+    apt-get install -y lua5.1 luarocks
+RUN luarocks install lua-cjson
+RUN luarocks install base64
```

```suggestion
RUN luarocks install dkjson
RUN luarocks install base64
```

If we use the `nickblah/lua:5.4-luarocks` image and the `dkjson` json library we won't need to use `apt-get` because `luarocks` is already installed.

---

Also, we should put the `RUN luarocks` **before** the `COPY ./ .` line (see #7  for examples) so that it doesn't have to reinstall dependencies every time source code changes.